ckanext/ontario_theme/templates/internal/header.html

```diff
@@ -87,7 +87,7 @@ <h1>
         {{ h.build_nav_main(
           ('search', _('Datasets')),
           ('organizations_index', _('Organizations')),
-          ('group_index', _('Groups')),          
+          ('group_index', _('Collections')),          
```

Comment:
This is one area we probably do want to hard code the change for so that our navigation works as we want it. But we should change the route to collections.

`('collection_index', _('Collections')),`

Reply:
agreed. change made.